src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -320,11 +321,19 @@ private void MethodChecks(MethodDeclarationSyntax syntax, Binder withTypeParamsB
 
                     if ((object)overriddenMethod != null)
                     {
-                        CustomModifierUtils.CopyMethodCustomModifiers(overriddenMethod, this, out _lazyReturnType, 
-                                                                      out _lazyCustomModifiers, 
+                        CustomModifierUtils.CopyMethodCustomModifiers(overriddenMethod, this, out _lazyReturnType,
```

Comment:
will keep the current behavior, and add a test to assert it.
Will also file a bug to make sure we decide whether we want to support overriding a ref-readonly method that has no modifier or not.

---
In reply to: [118420724](https://github.com/dotnet/roslyn/pull/19658#discussion_r118420724) [](ancestors = 118420724)